frontend/src/App.js

```diff
@@ -1,49 +1,45 @@
-import React, {useState} from 'react';
-import {
-    BrowserRouter as Router,
-    Switch,
-    Route,
-} from "react-router-dom";
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 // Import custom styles for our application
 import './App.css';
 
 import Auth from './services/Auth';
-import Navbar from "./components/layout/Navbar";
+import Navbar from './components/layout/Navbar';
 
 // Import pages
-import LoginPage from "./components/auth/LoginPage";
+import LoginPage from './components/auth/LoginPage';
 import HomePage from './components/home/HomePage';
-import PostsPage from "./components/posts/PostsPage";
+import PostsPage from './components/posts/PostsPage';
 import ChatPage from './components/chat/ChatPage';
 
 function App() {
   const [loggedIn, setLoggedIn] = useState(Auth.isLoggedIn());
   Auth.bindLoggedInStateSetter(setLoggedIn);
-  
+
   const loggedInRouter = (
-            <Router>
-                <Navbar onLogout={() => Auth.logout()} />
-
-                <div className="container mt-5">
-                    <Switch>
-                        <Route path="/posts">
-                            <PostsPage/>
-                        </Route>
-
-                        <Route path="/chat">
-                            <ChatPage/>
-                        </Route>
-
-                        <Route path="/">
-                          <HomePage/>
-                        </Route>
-                    </Switch>
-                </div>
-            </Router>
+    <Router>
+      <Navbar onLogout={() => Auth.logout()} />
+
+      <div className="container mt-5">
+        <Switch>
+          <Route path="/posts">
+            <PostsPage />
+          </Route>
+
+          <Route path="/chat">
+            <ChatPage />
+          </Route>
+
+          <Route path="/">
+            <HomePage />
+          </Route>
+        </Switch>
+      </div>
```

Comment:
Maybe Login page Route should also be added, otherwise React does not know the path to it...

Reply:
Login page is not routed. It is accessed by default in case the user in not logged in. See line 42 below.